idaes/core/base/tests/test_control_volume_0d_scaler_object.py

```diff
@@ -34,6 +34,10 @@
 from idaes.core.base.control_volume0d import ControlVolume0DScaler
 
 
+def approx(x):
+    return pytest.approx(x, rel=1e-15, abs=0)
```

Comment:
This tolerance is extremely tight, could there be failures in the future from this?

Reply:
Here we're trying to match scaling factors like `1/10 * 1/5 * 3/10`. Floating point arithmetic is not associative, so `(1/10 * 1/5)*(3/10) !=(1/10) * (1/5*3/10) `. However, it will be within a relative tolerance of machine epsilon, which is `2.22e-16` for double precision arithmetic in Python. So no, there shouldn't be failures in the future from this.